rfcs/text/0048-fileorigin-fields.md

```diff
@@ -82,6 +82,44 @@ Example sources of data is shown in the above.
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting, or if on the larger side, add them to the corresponding RFC folder.
 -->
 
+The following is the real world example and usage for these fields.
```

Comment:
If you have an example of how this can be used on macos, I think it would be good to include here too